commit c07d39a11a5a8b52a3f290274e8c664b9a0b14c2 (refs/changes/12/3212/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-01-12T14:24:12-08:00 (1 year, 9 months ago)
    
    TRITON-46 sdc-cloudapi volumes-automount.test.js can stall when waiting for container to be deleted

diff --git a/test/volumes-automount.test.js b/test/volumes-automount.test.js
index 671407d..6def9f6 100644
--- a/test/volumes-automount.test.js
+++ b/test/volumes-automount.test.js
@@ -494,7 +494,7 @@ if (CONFIG.experimental_cloudapi_nfs_shared_volumes !== true) {
                         t.ok(machine.state, 'should have state, got: '
                             + machine.state + ' (try ' + attempt + ')');
                         state = machine.state;
-                        if (state === 'deleted') {
+                        if (state === 'deleted' || state === 'failed') {
                             t.end();
                         } else {
                             setTimeout(getState, 5000);
